README.md

```diff
@@ -2,3 +2,23 @@
 
 > Curated list of awesomeness for Persistent Browser Based Games.
 
+[Resources](#resources)
+
+- [PBBG.com](https://pbbg.com) The home of PBBG's and text based browser games.
```

Comment:
What do you think of simplifying to "The home of Persistent Browser-Based Games"?  That way we're not inferring a bias towards text-based games.

Reply:
Completely up to you, the text I added was just added on a whim to get going so I'm all for this change :)